modules/lvm2/udiskslinuxvolumegroupobject.c

```diff
@@ -610,17 +611,20 @@ update_vg (GObject      *source_obj,
           if (cmp_int_lv_name ((*lvs_p2)->lv_name, lv_info->metadata_lv))
             meta_lv_info = *lvs_p2;
 
+      if (lv_info->pool_lv && g_strcmp0 (lv_info->segtype, "vdo") == 0)
+        vdo_info = bd_lvm_vdo_info (lv_info->vg_name, lv_info->pool_lv, &error);
```

Comment:
You should do something with the `error` set.

Reply:
Fixed (both locations).